### PR TITLE
Explain how to use the new dynamic config

### PR DESCRIPTION
**Pull request by peterrus**
Explain how to use the new dynamic config introduced in #25 and remove references to the old style. I noticed the docs were not in sync with how the configuration system works now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,33 +99,13 @@ docker build -t pleroma .
...
-Setup the database:
-
-```sh
-docker-compose run --rm web mix ecto.migrate
-```
-
-Get your web push keys and copy them to `secret.exs`:
-
-```
-docker-compose run --rm web mix web_push.gen.keypair
```
Comment (peterrus) [inline README.md:147]: This is done automatically in the new dynamic config afaik.

File: README.md [hunk 2]
```diff
@@ -200,6 +144,11 @@ docker build -t pleroma . --build-arg PLEROMA_VER=v2.0.7 # a version
 
 `a9203ab3` being the hash of the commit. (They're [here](https://git.pleroma.social/pleroma/pleroma/commits/develop))
 
+## Config Override
+By default the provided `docker-compose.yml` file mounts `config.exs` in the Pleroma container, this file is a dynamic configuration that sources some values from the environment variables provided to the container (variables like `ADMIN_EMAIL` etc.).
+
+For those that want to change configuration that is not exposed through environment variables there is the option to mount the `config-override.exs` file which can than be modified to your satisfaction. Values set in this file will override anything set in `config.exs`. The override file provided in this repository disables new registrations on your instance, as an example.
+
```
Comment (peterrus) [inline README.md:151]: I thought it'd be useful to have an override file so people don't have to modify the upstream `config.exs` in their local clones. The dynamic config already tries to include this file, so I just had to mount it into the container.

**Comment by peterrus** [comment]
@excodex @felix-two-tone Want to review (and hopefully merge) this for me? Much appreciated (:

**Comment by peterrus** [comment]
@excodex @felix-two-tone This builds on your earlier work so feel free to let me know if you have any opinions on this.

**Comment by felix-two-tone** [comment]
So yea, this answers some questions that I had about what was going on with the two configs involved (Guess I should read more on this repo). So question, does this part of the process even need to occur anymore? I didn't really understand in the README.md that it was asking you to create this config file, it seemed to me that it never did anything.

https://github.com/angristan/docker-pleroma/blob/09fa1e81e3a13c919732b2779e19e5cc7cae3d59/README.md?plain=1#L32

File: README.md (hunk 1 above)
Comment (felix-two-tone) [reply README.md:147]: I would run this religiously but never thought it did anything. Seems I was right.

**Comment by peterrus** [comment]
@felix-two-tone nope, you don't need to create that config file anymore as it is included in the repo already and sources some values from the environment variables. If you want to configure things that are not configurable through environment variables you can use the `config-override` I added in this PR.

I have also removed the instructions to create that `config/secret.exs` file as it does indeed not get used and thus does nothing. Just use the `config-override` or hack the included dynamic config if you're so inclined (but I would not recommend that because your changes will conflict with any changes to that file being done upstream).

**Comment by peterrus** [comment]
@angristan should be ready to merge unless you have any questions or remarks.

**Review by angristan — APPROVED** [review]

**Pull request merged**